sceptre/scipool/config/prod/sc-product-ec2-linux-docker-notebook.yaml

```diff
@@ -0,0 +1,19 @@
+template:
+  path: "sc-product-ec2-linux-docker-notebook.j2"
+stack_name: "sc-product-ec2-linux-docker-notebook"
```

Comment:
you'll need to add workaround here as well, https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/.github/workflows/aws-scipool.yaml#L54-L62

Reply:
@zaro0508 Why does `aws-scipool.yaml` only refer to `prod` and not to `develop`, `strides`, or `bmgfki`?

Reply:
we forgot to enable it for all prod accounts, the fix is in PR #1071.  it doesn't need to be setup for develop because we want to keep older versions around for debugging in develop account.